### PR TITLE
Instructions unclear... Not sure exactly 'Change It' button has been …

### PR DESCRIPTION
**Pull request by Superraptor**
…given no function. Furthermore, no idea if should display all of the result array or just one idea... Otherwise it is the same everytime. Additionally, I realize it's very difficult, almost impossible to create/read cookies in jQuery without the external library, so without using that library I'm not sure where to go. Also not sure if I can use Bootstrap or not, so I didn't. Basically all of the cookie related stuff should work in JS, but doesn't in jQuery, but since I've hit a brick wall and don't have any more time to work on it, this is as far as I could get...

### CODE REVIEW

**Comment by Superraptor** [comment]
Charles Kronk
cjk70@pitt.edu

**Comment by Superraptor** [comment]
Tried to add another commit because I just saw the updated instructions, but no sure if it worked?

File: template/quiz2.html [hunk 1]
```diff
@@ -1,8 +1,13 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
 	<head>
-		<meta charset="UTF-8"
+		<meta charset="UTF-8">
 		<title>Quiz 2</title>
+		
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
```
Comment (matty-digital) [inline template/quiz2.html]: scripts should go as close to the closing body tag as possible, when possible

File: template/quiz2.js [hunk 2]
```diff
@@ -1,39 +1,85 @@
...
 	// Do not commit console.log statements
 	// Use ajax to reach the api endpoint
 	// Whether or not you use jQuery, you still have to fix the jQuery errors. Removing jQuery is not fixing the problem.
+	"use strict";
+	$(document).ready(function () {
+		
+		var $mouseover = $('.mouseover'), $click = $('.click'), $submit = $('.submit'), $sendData = $('#sendData'), $changeIt = $('#changeIt'), $keepIt = $('#keepIt'), $prevData = [], $myCookie;
+		
+		console.log(document.cookie);
+		if (document.cookie.length > 0) {
```
Comment (matty-digital) [inline template/quiz2.js]: `document.cookie` is the way to go without a jQuery library

**Pull request closed without merging**